first_app/hello.py

```diff
@@ -117,39 +112,65 @@ def login_api():
     access_token = jwt.encode(token_data, SECRET, algorithm='HS256')
     return {"access_token": access_token}, 200
 
-# J
 
 @app.route("/api/v1/user-info/<user_id>", methods=['GET'])
 # @is_authenticated https://pythonworld.ru/osnovy/dekoratory.html
 # Треба створити декоратор, який буде дозволяти доступ до апі тільки залогіненим користувачам.
 @login_required
 def user_info_api(user_id):
-    db = get_db()
-    user = db.execute(
-        "SELECT * FROM user WHERE id = ?",
-        (user_id,),
-    ).fetchone()
+    user = User.query.filter(User.id == user_id).one()
     return {
         "id": user["id"],
         "phone_number": user["phone_number"],
         "first_name": user["first_name"],
         "second_name": user["second_name"],
     }, 200
-
+    
+    
+@app.route('/api/v1/user-posts/<user_id>', methods=['GET'])
+@login_required
+def create_post(user_id):
+    posts = Post.query.all()
+    posts = []
+    for post in posts:
+	posts.append({
+	"id": post["id"],
+	"title": post["title"],
+	"body": post["body"],
+	"created": post["created"],	
+	})    
+    return json(posts)
+    
+
+@app.route('/api/v1/edit-posts/<user_id>', methods=['GET'], ['POST'])
+def post_edit(user_id):
+    post = Post.query.get(user_id)
+    if request.method == "POST":
+        post.title = request.form['title']
+        post.body = request.form['body']
+        post.created = request.form['created']
+        db.session.commit()
+      #return redirect('/user-posts')
+            		        
+    return "Post edited", 200	
+    
+    
+@app.route('/api/v1/deleted-posts/<user_id>, methods=['GET'], ['POST'])
+ def delete_posts(user_id):
+    post = Post.query.get(user_id)
```

Comment:
This api will remove all users post. But, we usually remove (delete) only chosen post, right?